operator/src/main/java/oracle/kubernetes/operator/helpers/JobHelper.java

```diff
@@ -323,6 +323,7 @@ private String getAsServiceName() {
       addEnvVar(vars, IntrospectorJobEnvVars.WDT_DOMAIN_TYPE, getWdtDomainType());
       addEnvVar(vars, IntrospectorJobEnvVars.DOMAIN_SOURCE_TYPE, getDomainHomeSourceType().toString());
       addEnvVar(vars, IntrospectorJobEnvVars.ISTIO_ENABLED, Boolean.toString(isIstioEnabled()));
+      addEnvVar(vars, IntrospectorJobEnvVars.PORT_FORWARDING_ENABLED, Boolean.toString(isPortForwardingEnabled()));
```

Comment:
This will cause introspection runs and eventually roll the domain in upgrade situation.

Reply:
This environment variable is only added to the introspector job pod and not to the WLS server pods. So I think pods should not roll in case of an upgrade situation. I'll give it a try to confirm this.

Reply:
Oh, ok.

Reply:
Please specifically test this for Model-in-Image -- I suspect the MII introspector will detect the change in configuration and recommend a roll...

Reply:
I tested it for MII and am not seeing a roll. Since the env variable is added only in the job helper and the previous job will not be available for comparison, I think it'll not be able to detect the change in job configuration.

Reply:
I was also thinking about the configuration change. How about when the MII online-update `domain.spec.configuration.model.onlineUpdate` is set to `enabled=true` and `onNonDynamicChanges=CommitUpdateAndRoll`?